src/client/results/mostfrequent/users/MostActiveUsers.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import MostFrequent from './../MostFrequent';
+import { updateActiveUsersSearch } from './../mostFrequentActions';
+import UserItemsList from './UserItemsList';
+
+const MostActiveUsers = ({ dispatch, userInfoList, filterTerm }) => {
+  const filteredItems = userInfoList.filter(
+    (userInfo) => userInfo.author.name.toLowerCase().includes(filterTerm.toLowerCase())
+    || userInfo.author.handle.toLowerCase().includes(filterTerm.toLowerCase())
+  ).slice(0, 100);
```

Comment:
I'd consider adding explicit curly braces here as such:
```
const filteredItems = userInfoList.filter((userInfo) => {
    const matchName = userInfo.author.name.toLowerCase().includes(filterTerm.toLowerCase());
    const matchHandle = userInfo.author.handle.toLowerCase().includes(filterTerm.toLowerCase());

    return matchName || matchHandle;
})
```
to make it that bit clearer. Of course that means we ALWAYS evaluate the handle check, rather than only if the name check fails. Just a consideration, your call on which to use.